configurations/CAndy/dev/allowed-schema.csv

```diff
@@ -0,0 +1,2 @@
+author_did,schema_name,version,details
+<CDT DID TBD>,contractor-credential,1.0,https://github.com/bcgov/digital-trust-toolkit/blob/contractor-credential/docs/governance/employment/contractor-credential/governance.md#261-schema-definition
```

Comment:
Question for @Gavinok or @esune,
Does the details field support arbitrary text, such as the reference URL I've specified here?

Reply:
Yes, the details field is free text - I forget how long it can be though.